src/Components/Dropdown/Dropdown.tsx

```diff
@@ -60,12 +60,14 @@ const Dropdown: React.FC<Props> = ({
         freeSolo={enableAutoComplete}
         disableClearable={!enableAutoComplete}
         isOptionEqualToValue={(option, val) => option?.value === val?.value}
+        // sonarignore:start
```

Comment:
In the file, I noticed multiple instances of `// sonarignore:start` and `// sonarignore:end` comments. If the issues in this file are related to type problems, I would recommend taking a closer look into the code to provide appropriate types wherever possible. These comments should ideally be used as a last resort when it's not feasible to resolve the issues through proper typing.

Reply:
Yes, they are basically about parentheses around single parameter, but I have replaced no sonar with disable prettier comment